### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by marcodussin** [joelshepherd/tabliss#606]

**Comment by dimateos** [comment]
It certainly has some open PR and unanswered issues but managing this kind of projects is time consuming.
There was activity in January so I would not consider it dead.

**Comment by the-wright-jamie** [comment]
I disagree, the project itself has not been rebuilt or updated on the Chrome Store in ages (+1 year). Ideally, @joelshepherd would create a CI to build the project and automatically publish changes (if they no longer want anything to do with this project); have people merge into a testing branch (say a new branch like `stage` or something) and have the moderator @gonzalrick merge the testing branch into trunk once testing is done or it's time for a release.

I'd love to help set this up if either of you see this 🙃 I remember reading that Shepherd is busy with other work which is completely understandable, but there is clearly a lot of support for this project and it would be a shame to see it bit rot over the coming years if things continue the way they are going.

Either this or it's time to fork - I'd love to lead that, been looking to get my hands dirty in FOSS for a while.
